texts/en_US.properties

```diff
@@ -62,6 +59,15 @@ geyser.chat.secure_info_1=Bedrock players are likely unable to chat, as the serv
 geyser.chat.secure_info_2=See {0} for more information and details on how to allow chat.
 geyser.chat.too_long=Your message is bigger than 256 characters ({0}) so it has not been sent.
 
+geyser.command.bedrock_only=That command can only be executed by a Bedrock player!
+geyser.command.extension_desc=The main command for the {0} Geyser extension
+geyser.command.geyser_desc=The main command for Geyser
```

Comment:
why is it `geyser.commands.help.desc`, but here `geyser.command.geyser_desc`? (or `geyser.command.extension_desc`)  For consistency it would be probably a good idea to have the new one's follow the same pattern

Reply:
I mostly just went with the current form for three reasons I can think of:

- `geyser.commands.x` are all of Geyser's "subcommands" (`/geyser ...`), so the root description of `/extensionId`  would be arguably out of place

- As above, `geyser.command.geyser_desc` would be kind of a grey area since `/geyser` is not really a "subcommand" itself but redirects to `/geyser help`

- `geyser.command.geyser_desc` and `geyser.command.geyser_desc` are only the root descriptions sent to the bedrock client, whereas the others we speak of are all basically just used for the help menu.

What do you think of 
`geyser.command.root.geyser`
`geyser.command.root.extension`

Reply:
Valid points! And I like the suggestions :)

Reply:
cool :)